doc/COMPILING/COMPILING-CMAKE.md

```diff
@@ -94,7 +94,7 @@ Obtain packages specified above with your system package manager.
  is done using CMake itself, and the actual build is done using either `make`
  (for Makefiles generator) or build-system agnostic `cmake --build . ` .
 
- There are two ways to build CataclysmDDA with CMake: inside the source tree or
+ There are two ways to build Cataclysm-DDA with CMake: inside the source tree or
```

Comment:
I've seen the name abbreviated as "Cataclysm: DDA" on the wiki, but not the version with a hyphen.

Reply:
On the other hand, the docs in this repo use both "Cataclysm-DDA" and "Cataclysm: DDA". :shrug: